Cargo.toml

```diff
@@ -92,3 +92,11 @@ features = ["ansi", "env-filter", "fmt", "time"]
 lto = true
 panic = 'abort'
 codegen-units = 1
+
+[patch.crates-io]
+f4jumble = { git = "https://github.com/zcash/librustzcash.git", rev = "2519e16ad8ab3a6af59a49028e014e13dddfc36c" }
+zcash_address = { git = "https://github.com/zcash/librustzcash.git", rev = "2519e16ad8ab3a6af59a49028e014e13dddfc36c" }
+zcash_encoding = { git = "https://github.com/zcash/librustzcash.git", rev = "2519e16ad8ab3a6af59a49028e014e13dddfc36c" }
+zcash_note_encryption = { git = "https://github.com/zcash/librustzcash.git", rev = "2519e16ad8ab3a6af59a49028e014e13dddfc36c" }
+zcash_primitives = { git = "https://github.com/zcash/librustzcash.git", rev = "2519e16ad8ab3a6af59a49028e014e13dddfc36c" }
+zcash_proofs = { git = "https://github.com/zcash/librustzcash.git", rev = "2519e16ad8ab3a6af59a49028e014e13dddfc36c" }
```

Comment:
I checked that this hash matches the current last commit on https://github.com/zcash/librustzcash/pull/580.

Reply:
This is no longer the case. It will be sorted out when we update `zcash_proofs` to depend on bellman 0.13.1.